### ISSUE TITLE
Color function name and parameters differently

### ISSUE THREAD

**Issue by Zethson** [MrDogeBro/sphinx_rtd_dark_mode#12]
Hi,

I'll try to be more precise now :)
The dark theme colors the function names & the parameters with the same color.
![image](https://user-images.githubusercontent.com/21954664/119802307-bbefbc80-bede-11eb-8855-a8f5e78ea7b1.png)

I would prefer to have the function name maybe white and the parameters in blue or they other way around.
Example: 
![image](https://user-images.githubusercontent.com/21954664/119802632-ff4a2b00-bede-11eb-9472-a9ed5151503b.png)
Much more readable in my opinion.

Again, if you prefer options here instead of implementing this for everyone I would be more than fine with this.
Just thought that I would provide some more feedback.

**Comment by MrDogeBro** [comment]
No that's supposed to be white for the function name so I don't know why it is not for you.......

**Comment by MrDogeBro** [comment]
@Zethson Ok so that's not my issue I don't think. If you look at it in light mode, it is all one color there (blue — see screenshot below). In places where it is 2 different colors in light mode, it is also two different colors in dark mode.

![Cap 05-27-21 08 58 02](https://user-images.githubusercontent.com/35872177/119829962-ca7fa900-bec9-11eb-92cd-fbffc785308d.png)

**Comment by Zethson** [comment]
> @Zethson Ok so that's not my issue I don't think. If you look at it in light mode, it is all one color there (blue — see screenshot below). In places where it is 2 different colors in light mode, it is also two different colors in dark mode so that seems to be a part of RTD theme that they are the same color.
> 
> ![Cap 05-27-21 08 58 02](https://user-images.githubusercontent.com/35872177/119829962-ca7fa900-bec9-11eb-92cd-fbffc785308d.png)

You are right actually. I guess I or others overwrote this always.
So yeah, maybe this could also be a further option? If you don't want to support this feel free to close this issue.

Thanks!

**Comment by MrDogeBro** [comment]
Yeah I don't know. It may become an option in the future but its not high priority right now.
